app/database_etl/etl_main.py

```diff
@@ -254,13 +251,63 @@ def get_coords(place_name, place_type):
                 break
     return coords
 
+
 def add_latlng_to_df(place_type, place_type_name, df):
     df['coords'] = df[place_type_name].map(lambda a: get_coords(a, place_type))
     df['longitude'] = df['coords'].map(lambda a: a[0] if isinstance(a, list) else None)
     df['latitude'] = df['coords'].map(lambda a: a[1] if isinstance(a, list) else None)
     df = df.drop(columns=['coords'])
     return df
 
+
+def get_most_recent_pub_date_info(row):
```

Comment:
nit: maybe we can call this `get_most_recent_publication_info` because the function is scraping more than just the publication_date, but all the publication info at large